### PR TITLE
pages: added placeholder page on /tutorials before redirect

### PR DESCRIPTION
**Pull request by skoh7645**
## Checklist

- [x] commit message follows [commit guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md#commit-message-guidelines).
- [x] website builds and runs in production - for information on how to test locally go [here](https://github.com/appsody/website/blob/master/DEVELOPMENT.md).

## Description
Added a placeholder page on /tutorials before redirect. 
Informs the user they are being redirected and provide links if the user is never redirected

## Related Issues
Fixes https://github.com/appsody/website/issues/117

### CODE REVIEW

**Comment by mhamwala** [comment]
Thanks for the PR, tested this locally, doesn't seem to be redirecting, could you test this again?

File: src/components/myRedirect.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+class MyRedirect extends React.Component {
+  componentDidMount() {
+    window.location.replace("https://medium.com/appsody");
```
Comment (tomleah) [inline src/components/myRedirect.js]: We can make this component more versatile by using `this.props`.

Suggestion:  `window.location.replace(this.props.to);`

File: src/pages/tutorials.js [hunk 2]
```diff
@@ -1,24 +1,28 @@
...
                         </div>
                     </div>
-                </div>
-            </section>
+                </section>
+            </div>
         </div>
-    </div>
-
+        <Redirect/>
```
Comment (tomleah) [inline src/pages/tutorials.js]: We can make this component more versatile by using react properties

Suggestion: `<Redirect to="https://medium.com/appsody"/>`

File: src/components/myRedirect.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+class MyRedirect extends React.Component {
```
Comment (tomleah) [inline src/components/myRedirect.js]: nit: `MyRedirect` -> `Redirect`

**Pull request merged**